src/explore-education-statistics-common/src/modules/charts/util/getUnit.ts

```diff
@@ -0,0 +1,23 @@
+import { DataSetCategoryConfig } from '@common/modules/charts/util/getDataSetCategoryConfigs';
+
+const getUnit = (
```

Comment:
Would possibly consider calling this method `getUnitOrDefault` just to be explicit in the fact that it is falling back to a default value if the first value doesn't exist?

Reply:
I've made this change now - `getUnit` is now `getUnitOrDefault`